client/my-sites/hosting/staging-site-card/card-content/manage-staging-site-card-content.tsx

```diff
@@ -169,6 +189,7 @@ export const ManageStagingSiteCardContent = ( {
 						<LeftActionsContainer>
 							<ManageStagingSiteButton />
 							<ConfirmationPushChangesButton />
+							<ConfirmationPullChangesButton />
```

Comment:
We will adjust those under the design ticket but for now, could we move the Push and Pull buttons to the new line under the Manage Staging Site and Delete button to improve readability?

Reply:
Do you think this is ok? ( I like it less than before )
![2023-09-29T10:03:02,079582682+03:00](https://github.com/Automattic/wp-calypso/assets/497103/3ebd565e-69fb-4106-aecb-6ea713b16c0d)

Reply:
Yes, the previous one has all the buttons wrapped into two lines, and it feels broken.

In the final design, we will wrap the site icon, title, URL, and first two buttons in a block, and we will keep everything related to the Push/Pull actions below, so in your new screenshot we are slightly going into the desired direction.